debugger/src/test/kotlin/org/rust/debugger/lang/RsFrameTypeDecoratorTest.kt

```diff
@@ -18,6 +18,42 @@ class RsFrameTypeDecoratorTest {
         "&str"
     )
 
+    @Test
+    fun `test decorate msvc ptr_const str`() = checkDecorate(
+        "ptr_const$<str>",
```

Comment:
Is it possible to get this combination at all?
It seems it was just `str` previously and now it is `ptr_const$<str$>`

Reply:
It's possible